presto-main/src/main/java/com/facebook/presto/metadata/FunctionRegistry.java

```diff
@@ -364,44 +364,44 @@ public WindowFunctionSupplier load(SpecializedFunctionKey key)
                 .aggregate(CovarianceAggregation.class)
                 .aggregate(RegressionAggregation.class)
                 .aggregate(CorrelationAggregation.class)
-                .scalar(SequenceFunction.class)
```

Comment:
I thought this was a pure rename. I didn't realize there were two different versions of the function.